lib/client/connection.js

```diff
@@ -305,8 +285,6 @@ Connection.prototype._handleBulkMessage = function(err, message, method) {
 };
 
 Connection.prototype._reset = function() {
-  this.seq = 1;
-  this.id = null;
```

Comment:
Note that I've deleted this (so `id` is never `null` after initialisation). The only place this seems to matter is in `Doc` (which I've fixed). I know we spoke about leaving this as-is, but it seems a little bit "messy" to stash the id in another property just to put it back here? I'm not sure I see the need.

That said, is this potentially a breaking change? ie might consumers _assume_ this behaviour on a disconnect?!

Reply:
As discussed, let's just ignore the idea of this being "breaking"; it's arguably an implementation concern.